controller/deploy/microshift-bootc/config-svc/update-banner.sh

```
@@ -0,0 +1,32 @@
+#!/bin/env python3
```

Shellcheck (0.11.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)